servicepulse/intro-failed-messages.md

```diff
@@ -65,37 +65,54 @@ The following actions can be performed on a failed message group:
+Both of these message list views allow for taking actions on an individual message, on custom message selections or on all messages contained in the view.
 
 NOTE: Retrying one or a few individual messages can be useful for testing system fixes before deciding to retry several messages in a group. This is because retrying several messages take a long time and queue other ServiceControl operations for longer than desired.
 
+The following actions can also be taken on each individual message or selection of messages:
+
+* **Retry** - Sends message(s) to be reprocessed by the corresponding endpoint.
+* **Archive** - Archives message(s).
+
+
+### Message details page
+
+As of ServicePulse v1.8.0, each individual message can be browsed in order to see in-depth details about a given failed message, archive or retry that message.
+
+![Failed Messages Page](images/failed-message-page.png 'width=500')
+
+Individual messages can be accessed by clicking the respective entry in any of the message list views.
+
+Each invidual failed message view allows for viewing:
+
+* **Message metadata** - Failure timestamp, endpoint name and location, retry status.
+* **StackTrace** - Full .NET exception stacktrace.
+* **Headers** - Complete set of message headers.
+* **Body** - Serialized message body.
+
+The following actions can also be taken on any given message:
 
-##### Managing individual failed messages
+* **Retry** - Sends message to be reprocessed by the corresponding endpoint.
+* **Archive** - Archives the message.
+* **View in ServiceInsight** - Launches [ServiceInsight](/serviceinsight/), focusing on the failed message for in-depth analysis of the failure causes. This only works if ServiceInsight is installed on the local machine.
 
-The following actions can be taken on each individual message:
+#### Sharing message data from ServicePulse
 
- * **Message Details:** For a given failed message, displays the message type, exception description, endpoint name and location, and failure timestamp.
- * **StackTrace:** Displays the full .NET exception stacktrace.
- * **Headers:** Displays a complete set of message headers.
- * **Body:** Displays the serialized message body.
- * **Copy Message Id:** Copies the failed message unique ID to the clipboard.
- * **Open in ServiceInsight:** Launches [ServiceInsight](/serviceinsight/), focusing on the failed message for in-depth analysis of the failure causes. This only works if ServiceInsight is installed on the local machine.
+The URL from that message's page be copied to share the details of a specific message from ServicePulse.
 
 
-### Archived Messages tab
+## Archived Messages tab
 
-Failed messages that cannot be processed successfully (or should not be retried due to various application-specific reasons) can be archived.
+Failed messages that cannot be processed successfully (or should not be retried due to various application-specific reasons) can be archived and later viewed in the Archived Messages tab.
```

Comment:
>(or should not be retried due to various application-specific reasons)

is it should or could? I am not sure we have the right context to say that it should not be retried?